.github/workflows/update.yml

```
@@ -4,18 +4,48 @@ on:
+    strategy:
+      matrix:
+        runner:
+          - ubuntu-latest
+        deno:
+          - version: 2.x
+        node-version:
+          - "lts/*"
+    runs-on: ${{ matrix.runner }}
     steps:
       - uses: actions/checkout@v4
+        ref: master
```

actionlint (1.7.7)

unexpected key "ref" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)